presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/cli/TestRun.java

```diff
@@ -165,6 +165,11 @@ private Environment getEnvironment()
                                 .add(Integer.toString(TESTS_READY_PORT))
                                 .add(
                                         "java",
+                                        "-Xmx1g",
+                                        // Force Parallel GC to ensure MaxHeapFreeRatio is respected
+                                        "-XX:+UseParallelGC",
+                                        "-XX:MinHeapFreeRatio=10",
+                                        "-XX:MaxHeapFreeRatio=10",
```

Comment:
I think it should be possible to overwrite this externally.

Reply:
Which ones?

Reply:
`"-Xmx1g",`

Reply:
discussed offline: let's revisit when need arises